Libraries/Components/View/View.js

```diff
@@ -27,12 +27,94 @@ const View: React.AbstractComponent<
   ViewProps,
   React.ElementRef<typeof ViewNativeComponent>,
 > = React.forwardRef(
-  ({tabIndex, focusable, ...otherProps}: ViewProps, forwardedRef) => {
+  (
+    {tabIndex, focusable, role, accessibilityRole, ...otherProps}: ViewProps,
+    forwardedRef,
+  ) => {
+    let restProps = {...otherProps};
+
+    // Map role values to AccessibilityRole values
+    const roleToAccessibilityRoleMapping = {
```

Comment:
Please alpha-sort this

Reply:
Should this mapping be declared outside the View component scope?